src/vsr/superblock.zig

```diff
@@ -1413,6 +1413,7 @@ pub fn SuperBlockType(comptime Storage: type) type {
                 superblock.verify_manifest_blocks_are_acquired_in_free_set();
 
                 // TODO Repair any impaired copies before we continue.
+                context.copy = 0;
```

Comment:
A good one!

Don't see a super great way to make this bug jump out, only something relatively heavy-weigh like

```zig
            copy: ?union(enum) {
                cient_table: u8,
                free_set: u8,
            } = null,
```